content/en/agent/faq/linux-agent-2022-key-rotation.md

```diff
@@ -117,6 +119,28 @@ Alternatively, check if your `datadog.repo` file contains `https://keys.datadogh
 {{% /tab %}}
 {{< /tabs >}}
 
+## The datadog-signing-keys package
+
+<div class="alert alert-info"><strong>Note</strong>: this section only applies to DEB Agent package users.</div>
+
+Since Agent 6.31.0/7.31.0, all Datadog DEB packages have a soft dependency on the `datadog-signing-keys` package. Upon installation, this package:
+
+- Configures APT keys in the `/usr/share/keyrings/datadog-archive-keyring.gpg` keyring and also via `apt-key` when necessary. **This ensures that the upcoming APT repository signing key is trusted.**
+- Sets up a [debsig-verify policy][13] for Datadog packages. This allows you to verify signatures of individual DEB packages locally. For example, to verify if a locally downloaded DEB package was really built and signed by Datadog, you can run the following command:
+
+  ```bash
+  $ debsig-verify datadog-dogstatsd_7.34.0-1_amd64.deb
+  ```
+
+  If the verification is successful, `debsig-verify` exits with status `0` and prints a message `debsig: Verified package from 'Datadog, Inc.' (Datadog).`. Note that Datadog's DEB packages embed signatures since version 6.26.0/7.26.0, so this won't work on earlier versions.
+
+Because the dependency of Agent 6.31.0+/7.31.0+ packages on the `datadog-signing-keys` package is optional, it might not get installed when:
+
+- You manually download the Agent DEB package and install it without having the Datadog repository configured as an APT source.
+- You mirror the Agent DEB package to your own APT repository without also mirroring the `datadog-signing-keys` package.
```

Comment:
```suggestion
- You mirror the Agent DEB package to your own APT repository without also mirroring the `datadog-signing-keys` package.
- Your APT configuration is set to not install _recommended_ packages, for example by running `apt` with ` --no-install-recommends` or by having `APT::Install-Recommends "0"` in `apt.conf`.
```

Reply:
I agree with adding this, but then I'll also need to modify the paragraph below, because this is the one option where it might actually cause a problem.